### DISCUSSION TITLE
Can't log in to a TEAMMATES instance

### DISCUSSION THREAD

**Discussion by ThomasGreen123** [TEAMMATES/teammates#12608] [Help requests] [answered]
### Environment

- **Commit SHA:** 
- **Operating System:** wins
- **JDK Version:** 17
- **Node.js Version:**
- **Firefox / Chrome Version (if applicable):**

### Resolution Attempt

I am currently working on issue https://github.com/TEAMMATES/teammates/issues/12588 to Improve the test code coverage of CommentRowComponent. I try to log in to a TEAMMATES instance first to learn more about the program,but it says 'you are not authorized to view the page'. I've only changed app.admins= to my Gmail in 'build.properties'. I am new to this project, is there anything I missed? Thanks a lot. 

![图片](https://github.com/TEAMMATES/teammates/assets/141736012/8e173ec9-0b69-4278-a659-d916165c230f)

**Comment by Tejasker** [comment]
It's the same issue with me too, I am able to fire up the dev server & angular server but I am not able to login into teammates.
I too need help in creating a TEAMMATES instance, I have gone through the documentation & followed as instructed but I am not able to create an account TEAMMATES instance

**Reply by weiquu** [reply, marked as answer]
Hi @Tejasker, can I check if you've followed the steps under the "[Logging in to a TEAMMATES instance](https://teammates.github.io/teammates/development.html#logging-in-to-a-teammates-instance)" section? Do also ensure that **both servers** and your **database** instance are running.

If you have, do try logging in as an administrator first with the instructions and let us know what is the error (e.g. in the console or on the dev server). Do also make sure your `build.properties` file is configured correctly.

**Reply by Tejasker** [reply]
Hii @weiquu 
Actually, in my case, the database instance is not running & this led me to stop creating admin access in the website and further the instructor roles too as said by you, I have managed to run all the 3 instances 
1. Both Servers (Front-end & back-end)
2. Local database emulator
now I can have all the functionalities
